app/src/main/java/mozilla/lockbox/store/RouteStore.kt

```diff
@@ -32,18 +43,48 @@ class RouteStore(
             .filterByType(RouteAction::class.java)
             .subscribe(routes as Subject)
 
+        this.onboarding
```

Comment:
phew! this is all pretty hairy; I'd love to take some time this afternoon to talk to you about the routing approach here.